### PR TITLE
Update notification templates

### PR DESCRIPTION
**Pull request by sh31**
Trello #1039
Google doc describing changes: https://docs.google.com/document/d/1tSoL2Cbal8kJvP-tgINmVr1rRgvFgDbXhctwFr_-low/edit

### CODE REVIEW

File: vulnerable_people_form/templates/_spl_match_email_template.md [hunk 1]
```diff
@@ -1,18 +1,20 @@
 Dear {{first_name}} {{last_name}},
 
-Your registration number is {{reference_number}}.
+Your registration number is {{submission_id}}.
```
Comment (sanisc) [inline vulnerable_people_form/templates/_spl_match_email_template.md:3]: has this changed ?

File: vulnerable_people_form/templates/_spl_match_sms_template.txt [hunk 2]
```diff
@@ -1,15 +1,15 @@
 Dear {{first_name}} {{last_name}},
 
-Your registration number is {{reference_number}}.
+Your registration number is {{submission_id}}.
```
Comment (sanisc) [inline vulnerable_people_form/templates/_spl_match_sms_template.txt:3]: same as above

File: vulnerable_people_form/templates/_spl_match_email_template.md (hunk 1 above)
Comment (sh31) [reply vulnerable_people_form/templates/_spl_match_email_template.md]: Copy-paste error. It's changed in the backend only.

**Review by sanisc — APPROVED** [review]

**Comment by sanisc** [comment]
LGTM

**Pull request merged**